core/new-gui/src/app/dashboard/component/feature-container/user-project-list/user-project-list.component.ts

```diff
@@ -51,20 +51,26 @@ export class UserProjectListComponent implements OnInit {
     this.userProjectEntriesIsEditingName = this.userProjectEntriesIsEditingName.filter(index => index != pid);
   }
 
-  public saveProjectName(project: UserProject, newName: string, index: number): void {
+  public saveProjectName(project: UserProject, newName: string): void {
     // nothing happens if name is the same
     if (project.name === newName) {
       this.removeEditStatus(project.pid);
     } else if (this.isValidNewProjectName(newName, project)) {
       this.userProjectService
       .updateProjectName(project.pid, newName)
       .pipe(untilDestroyed(this))
-      .subscribe(() => {
-        this.removeEditStatus(project.pid);
-        this.getUserProjectArray(); // refresh list of projects, name is read only property so can"t edit
-      });
+      .subscribe(
+        () => {
+          this.removeEditStatus(project.pid);
+          this.getUserProjectArray(); // refresh list of projects, name is read only property so cannot edit
+        },
+        (err: unknown) => {
+          // @ts-ignore
+          this.notificationService.error(err.error.message);
+        }
+      );
     } else {
-      // show error message and don"t call backend
```

Comment:
I recall there were other `"` typos in the code, can you double check other occurrences?

Reply:
yes, there was one other occurrence in this file that I had removed.  I also checked all other files I had created / modified just in case